_posts/2020-11-31-homebrew-2.6.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 2.6.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
+
+Major changes and deprecations since 2.5.0:
+
+- [macOS Big Sur is supported (and High Sierra unsupported)](https://github.com/Homebrew/brew/pull/9054)
+- [`depends_on :java`, `brew switch`, `brew diy` and various other APIs have been deprecated](https://github.com/Homebrew/brew/pull/9209)
+- [All `brew cask` commands have been deprecated in favour of `brew` commands (with `--cask`) when necessary](https://github.com/Homebrew/brew/pull/8899)
+- [`Homebrew.args` is deprecated](https://github.com/Homebrew/brew/pull/8795)
+- [All `Requirement`s are deprecated in Homebrew/core](https://github.com/Homebrew/brew/pull/9052)
+- [macOS Homebrew on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side).](https://github.com/Homebrew/brew/pull/9117)
```

Comment:
```suggestion
- [macOS Homebrew on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side)](https://github.com/Homebrew/brew/pull/9117).
```

Reply:
```suggestion
- [macOS Homebrew running natively on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side)](https://github.com/Homebrew/brew/pull/9117). We currently recommend running Homebrew using Intel emulation with Rosetta 2.
```